<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConditionWidget</class>
 <widget class="QWidget" name="ConditionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Condition</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QLabel" name="positiveLabel">
     <property name="text">
      <string>Positive</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="deadBandLabel">
     <property name="text">
      <string>Dead Band</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="offsetLabel">
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="negativeLabel">
     <property name="text">
      <string>Negative</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="coefficientLabel">
     <property name="text">
      <string>Coefficient</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="saturationLabel">
     <property name="text">
      <string>Saturation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="bothAxesCheckBox">
     <property name="text">
      <string>Both Axes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="symmetricCheckBox">
     <property name="text">
      <string>Symmetric</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="negativeCoefficientSpinBox"/>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="positiveCoefficientSpinBox"/>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="negativeSaturationSpinBox"/>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="positiveSaturationSpinBox"/>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="deadBandSpinBox"/>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="offsetSpinBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
